File: Fire drill 1/Java/TaskFive.java
public class TaskFive {
	public static void main(String[] args) {
		for(int num = 1; num <= 10; num++) {
			if(num % 4 == 0) {
				for(int count = 1; count <= 5; count++) {
					System.out.print(num);
				}
				System.out.print(" ");
			}
		}
	}
}

File: Fire drill 1/Javascript/TaskFive.js
for(let num = 1; num <= 10; num++) {
	if(num % 4 == 0) {
		for(let count = 1; count <= 5; count++) {
			process.stdout.write(String(num));
		}
		process.stdout.write(` `);
	}
}